test/unit/system_database_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class SystemDatabaseTest < ActiveSupport::TestCase
+
+  def test_configuration_specification_is_memoized
+    FakeFS do
+      expected_conf = System::Database.configuration_specification
+      assert_instance_of ActiveRecord::ConnectionAdapters::ConnectionSpecification, expected_conf
+      config = Rails.root.join('config', 'database.yml')
```

Comment:
Tiny thing, but can we please rename this to `config_path` or `config_file`?

Reply:
`config_path` looks better to me because we don't have a config or a file itself, we have just a path there.